apps/vast/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG VAST_VERSION=latest
```

Comment:
Would it make sense to add VAST as a submodule to avoid this silently breaking CI?

Also, since `pyvast-threatbus` integrates with the proprietary VAST matcher plugin, does it even make sense to use latest open-source VAST? Should this Dockerfile instead live alongside our closed-source plugins?

Reply:
As discussed via a call, we will use the latest VAST release version and update it manually as part of releasing Threat Bus.